src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/gtfs/ScheduleTestsIT.java

```diff
@@ -0,0 +1,132 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.gtfs;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertEquals;
+
+import java.time.ZoneId;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.jayway.restassured.http.ContentType;
+
+import fi.otavanopisto.kuntaapi.server.integrations.gtfs.GtfsConsts;
+import fi.otavanopisto.kuntaapi.server.integrations.ptv.PtvConsts;
+import fi.otavanopisto.kuntaapi.test.AbstractIntegrationTest;
+
+public class ScheduleTestsIT extends AbstractIntegrationTest{
+  
+  private static final ZoneId TIMEZONE_ID = ZoneId.of("Europe/Helsinki");
+  
+  /**
+   * Starts WireMock
+   */
+  @Rule
+  public WireMockRule wireMockRule = new WireMockRule(getWireMockPort());
+  
+  @Before
+  public void beforeTest() throws InterruptedException {
+    createPtvSettings();
+    
+    getPtvMocker()
+      .mockOrganizations("0de268cf-1ea1-4719-8a6e-1150933b6b9e")
+      .startMock();
+
+    waitApiListCount("/organizations", 1);
+    
+    createGtfsSettings(getOrganizationId(0));
+
+    waitApiListCount(String.format("/organizations/%s/transportSchedules", getOrganizationId(0)), 17); 
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "/organizations/%s/transportSchedules" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)